### ISSUE TITLE
Training data used with saved model

### ISSUE THREAD

**Issue by roya0045** [remicres/sr4rs#27]
Greetings. I'm curious to know if you could share more insight on the training dataset and methodology. 

I would guess that the High-resolution dataset and the lower resolution one were taken on the same day? 

Were spatial and other temporal (such as mentionned previously) criterions taken into account with the training dataset to have the model generalize well accross the globe?

**Comment by remicres** [comment]
Hi @roya0045 

> I would guess that the High-resolution dataset and the lower resolution one were taken on the same day?
Yes. The provided pre-trained model has been trained over a single (Spot-7, Sentinel-2) image couple.

**Comment by quizz0n** [comment]
Hi, 
I'm also interested in this topic. Are there any limitations regarding the source of HR image? In your example you mention NIR band, can Orthoimagery be used also?

**Comment by remicres** [comment]
Hi,

Yes, you can use any source and/or target, with any number of spectral bands. Theoretically you can even train a deep net that uses source and/or target of different modality (e.g. SAR at input, optical at output)!
So yes, an ortho image would totally be a "good" target.

Just keep in mind that the perceptual loss (The VGG loss) applies only on the first 3 channels, and it intended to be used with RGB target images.
If your target image does not have RGB in first channels, it might be better to not use perceptual loss.

The case of (source, target) = (Sentinel-2, Spot-6/7) is friendly because the spectral bands are close.
